manifest.php

```diff
@@ -29,7 +29,7 @@
     'label' => 'QTI item model',
     'description' => 'TAO QTI item model',
     'license' => 'GPL-2.0',
-    'version' => '6.14.0',
+    'version' => '6.15.0',
```

Comment:
if it is a bug fix, should be 6.14.1... unless there are some new features together with this bug fix ?